axiom/nr_segment.h

```diff
@@ -117,6 +117,10 @@ typedef struct _nr_segment_metric_t {
 typedef struct _nr_segment_error_t {
   char* error_message; /* The error message that will appear on a span event. */
   char* error_class;   /* The error class that will appear on a span event. */
+  char* error_file;
```

Comment:
Can you add a comment for each of these fields like the existing ones have?

Reply:
Added: [eb64129](https://github.com/newrelic/newrelic-php-agent/pull/942/commits/eb6412921bd5a5b301a6f76ffde87aa581ce664e)